verifier/testreport.py

```diff
@@ -586,6 +586,15 @@ def characterize_results_by_options(self, test_list, category):
         # User self.failing_tests, looking at options
         results = defaultdict(lambda : defaultdict(list))
         results['locale'] = {}  # Dictionary of labels for each locale
+
+        # Look at particular test types
+        if test_list:
```

Comment:
why is there `if test_list` for plural_rules, but not for datetime_fmt?

Reply:
Good catch.